### PR TITLE
Pass mode i18n strings to Itinerary body

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**
Passes translated mode strings to itinerary body in otp-ui. Goes with https://github.com/opentripplanner/otp-ui/pull/549

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/narrative/line-itin/connected-itinerary-body.js
Comment (miles-grant-ibigroup) [inline lib/components/narrative/line-itin/connected-itinerary-body.js:89]: I think you forgot to import this!

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Code looks good, will leave blocked until I can test once the otp-ui pr is merged

File: lib/components/narrative/line-itin/connected-itinerary-body.js
Comment (amy-corson-ibigroup) [reply lib/components/narrative/line-itin/connected-itinerary-body.js:89]: Thank you!

**Pull request closed without merging**
